mindsdb/integrations/handlers/merlion_handler/requirements.txt

```diff
@@ -0,0 +1,5 @@
+mindsdb>=22.6.2.1
+mindsdb_sql >= 0.4.0
+pandas <=1.4.3
+
+1.2.0 <= salesforce-merlion <= 1.3.1
```

Comment:
Is it right format? My pip shows exception

Reply:
fixed